packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -169,86 +200,100 @@ export const ReleaseDetailsLayout = ({
-                    </PopoverButton>
-                  </Flex>
-                  <ReleaseInfoWrapper
-                    direction="column"
-                    justifyContent="center"
-                    alignItems="flex-start"
-                    gap={1}
-                    padding={5}
+                      <Typography variant="pi" color="neutral300">
+                        <RelativeTime timestamp={new Date(release.createdAt)} />
+                        {formatMessage(
+                          {
+                            id: 'content-releases.header.actions.created.description',
+                            defaultMessage: ' by {createdBy}',
+                          },
+                          { createdBy }
+                        )}
+                      </Typography>
+                    </ReleaseInfoWrapper>
+                  </Popover>
+                )}
+                <Button size="S" variant="tertiary">
+                  {formatMessage({
+                    id: 'content-releases.header.actions.refresh',
+                    defaultMessage: 'Refresh',
+                  })}
+                </Button>
+                <CheckPermissions permissions={PERMISSIONS.publish}>
+                  <Button
+                    size="S"
+                    variant="default"
+                    onClick={handlePublishRelease}
+                    loading={isPublishing}
                   >
-                    <Typography variant="pi" fontWeight="bold">
-                      {formatMessage({
-                        id: 'content-releases.header.actions.created',
-                        defaultMessage: 'Created',
-                      })}
-                    </Typography>
-                    <Typography variant="pi" color="neutral300">
-                      <RelativeTime timestamp={new Date(release.createdAt)} />
-                      {formatMessage(
-                        {
-                          id: 'content-releases.header.actions.created.description',
-                          defaultMessage: ' by {createdBy}',
-                        },
-                        { createdBy }
-                      )}
-                    </Typography>
-                  </ReleaseInfoWrapper>
-                </Popover>
-              )}
-              <Button size="S" variant="tertiary">
-                {formatMessage({
-                  id: 'content-releases.header.actions.refresh',
-                  defaultMessage: 'Refresh',
-                })}
-              </Button>
-              <Button size="S" variant="default">
-                {formatMessage({
-                  id: 'content-releases.header.actions.release',
-                  defaultMessage: 'Release',
-                })}
-              </Button>
-            </Flex>
+                    {formatMessage({
+                      id: 'content-releases.header.actions.release',
+                      defaultMessage: 'Release',
+                    })}
```

Comment:
I think we agreed and the designs have been updated with Publish now

Reply:
You're right, completely forgot about it, thanks!